test/lib/lbt/analyzer/FioriElementsAnalyzer.js

```diff
@@ -1,7 +1,7 @@
 const test = require("ava");
 const FioriElementsAnalyzer = require("../../../../lib/lbt/analyzer/FioriElementsAnalyzer");
 const sinon = require("sinon");
-const esprima = require("esprima");
+const parseJS = require("../../../../lib/lbt/utils/parseUtils");
```

Comment:
`parseUtils`

Reply:
Done in https://github.com/SAP/ui5-builder/pull/619